src/source/Rtp/RtpPacket.c

```diff
@@ -295,26 +295,29 @@ STATUS setBytesFromRtpPacket(PRtpPacket pRtpPacket, PBYTE pRawPacket, UINT32 pac
     }
     pCurPtr++;
 
-    putInt16((PINT16) pCurPtr, pHeader->sequenceNumber);
+    //https://tools.ietf.org/html/rfc7741#page-5
+    //All integer fields in the specifications are encoded as
+    //   unsigned integers in network octet order.
+    putUnalignedInt16BigEndian((PINT16) pCurPtr, pHeader->sequenceNumber);
```

Comment:
You can drop the casting here as these functions take PVOID. Also, consider if you want to use the macro version PUT_UNALIGNED_BIG_ENDIAN in which case you will need the casting

Reply:
makes sense

Reply:
@zhuker I was wondering whether you are going to make these changes so we can merge the PR?

Reply:
@MushMal done